src/main/java/com/epam/reportportal/cucumber/AbstractReporter.java

```diff
@@ -251,8 +251,8 @@ protected StartTestItemRQ buildStartScenarioRequest(TestCase testCase, String na
 	 * @param featureContext  current feature context
 	 * @param scenarioContext current scenario context
 	 */
-	protected void beforeScenario(RunningContext.FeatureContext featureContext, RunningContext.ScenarioContext scenarioContext) {
-		String scenarioName = Utils.buildName(scenarioContext.getKeyword(), AbstractReporter.COLON_INFIX, scenarioContext.getName());
+	protected void beforeScenario(RunningContext.FeatureContext featureContext, RunningContext.ScenarioContext scenarioContext, TestCase testCase) {
```

Comment:
Please do not change interface. Pick `TestCase` from `ScenarioContext` object.

Reply:
Thanks for that! I hadn't seen that `TestCase` could be accessed in a more straight-forward way. I corrected this now.